pkg/distributors/test/Exiter.test.ts

```diff
@@ -0,0 +1,98 @@
+import { ethers } from 'hardhat';
+import { Contract } from 'ethers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+
+import Token from '@balancer-labs/v2-helpers/src/models/tokens/Token';
+import TokenList from '@balancer-labs/v2-helpers/src/models/tokens/TokenList';
+
+import { bn, fp } from '@balancer-labs/v2-helpers/src/numbers';
+
+import { deploy } from '@balancer-labs/v2-helpers/src/contract';
+import * as expectEvent from '@balancer-labs/v2-helpers/src/test/expectEvent';
+import { expectBalanceChange } from '@balancer-labs/v2-helpers/src/test/tokenBalance';
+import { advanceTime } from '@balancer-labs/v2-helpers/src/time';
+import { setup, rewardsDuration } from './MultiRewardsSharedSetup';
+
+describe('Exiter', () => {
+  let lp: SignerWithAddress, mockAssetManager: SignerWithAddress;
+
+  let poolTokens: TokenList;
+  let vault: Contract;
+  let stakingContract: Contract;
+  let callbackContract: Contract;
+  let rewardToken: Token;
+  let pool: Contract;
+
+  before('deploy base contracts', async () => {
+    [, , lp, mockAssetManager] = await ethers.getSigners();
```

Comment:
```suggestion
    [, , lp, rewarder] = await ethers.getSigners();
```

afaict this isn't actually an asset manager (as it's explicitly allowlisted as a rewarder) plus this is more descriptive

Reply:
It looks like we've updated this in some of the other files but not here